modules/consensus/consensusset.go

```diff
@@ -23,6 +23,11 @@ var (
 	errNilGateway = errors.New("cannot have a nil gateway as input")
 )
 
+const (
+	sendHeaders = "SendHdrs"
+	sendHeader  = "SndHdr"
+)
+
```

Comment:
For the sake of easier reading and consistency, let's change the const names here to `sendHeadersCmd`, and `sendHeaderCmd`, and also add a `sendBlocksCmd` and `sendBlockCmd`.